common.blocks/section/_type/section_type_values.css

```diff
@@ -61,3 +61,91 @@
     background-position: 15px 15px;
     background-image: url(dl__dt_type_support.png);
 }
+
+@media screen and (max-width: 773px) {
+    .section__values {
+        width: 80%;
+        margin: 0 auto;
+    }
+
+    .section__values .dl__dd {
```

Comment:
last time we spoke we decided that `dl` was a bad idea and should be replaced with `div`. why you changed your mind?